server/hotrod/src/main/java/org/infinispan/server/hotrod/command/tx/AbstractForwardTxCommand.java

```diff
@@ -35,9 +35,4 @@ abstract class AbstractForwardTxCommand extends BaseRpcCommand {
    public boolean isReturnValueExpected() {
       return false;
    }
-
-   @Override
-   public boolean canBlock() {
```

Comment:
`ForwardCommitCommand` and `ForwardRollbackCommand` need to use the `BlockingManager`.